ts/src/base/Exchange.ts

```diff
@@ -5171,10 +5171,10 @@ export default class Exchange {
         const first = this.safeValue (result, 0);
         if (first !== undefined) {
             if ('timestamp' in first) {
-                return this.sortBy (result, 'timestamp');
+                return this.sortBy (result, 'timestamp', true);
```

Comment:
@pcriadoperez why do you think it should be descending here?

Reply:
I think usually when we return any array, may it be withdrawals, ohlcv.. with timestamps the first value is usually the latest and most recent. So I believe pagination results should be the same. Also it is something we test for.